pkgs/applications/networking/irc/thelounge/default.nix

```diff
@@ -51,6 +66,13 @@ stdenv.mkDerivation (finalAttrs: {
   preInstall = ''
     rm -rf node_modules
```

Comment:
> Would the sqlite building still be necessary if node_modules was not removed?

Yes, since we initially pass `--ignore-scripts`.

> Yarn should prune during install anyway so removing shouldn't be necessary

Thanks, I'll give it a shot.